### PR TITLE
Added spanish translation of quicktour.mdx

### PR DESCRIPTION
**Pull request by duedme**
# Translation of quicktour.mdx into Spanish

I made the translation of quicktour.mdx into Spanish (fixes #15947). The document is located in the transformers/docs/source_es folder.

@omarespejel 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?

### CODE REVIEW

File: docs/source_es/quicktour.mdx [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+
+```py
+>>> results = classifier(["We are very happy to show you the 🤗 Transformers library.", "We hope you don't hate it."])
+>>> for result in results:
+...     print(f"label: {result['label']}, with score: {round(result['score'], 4)}")
+label: POSITIVE, with score: 0.9998
+label: NEGATIVE, with score: 0.5309
+```
+
+El [`pipeline`] también puede iterar sobre dataset entero. Comienza instalando la biblioteca [🤗 Datasets](https://huggingface.co/docs/datasets/):
```
Comment (omarespejel) [inline docs/source_es/quicktour.mdx:96]:
```suggestion
El [`pipeline`] también puede iterar sobre un dataset entero. Comienza instalando la biblioteca [🤗 Datasets](https://huggingface.co/docs/datasets/):
```

File: docs/source_es/quicktour.mdx [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+```
+
+El pipeline descarga y almacena en caché un [modelo preentrenado](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english) por defecto y tokeniza para análisis de sentimiento. Ahora puedes usar `classifier` en tu texto objetivo:
+
+```py
+>>> classifier("We are very happy to show you the 🤗 Transformers library.")
+[{'label': 'POSITIVE', 'score': 0.9998}]
+```
+
+Para más de una oración, entrega una lista de frases al [`pipeline`], que devolverá una lista de diccionarios:
```
Comment (omarespejel) [inline docs/source_es/quicktour.mdx:86]:
```suggestion
Para más de un enunciado entrega una lista de frases al [`pipeline`] que devolverá una lista de diccionarios:
```

File: docs/source_es/quicktour.mdx [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+* Traducción: traduce un texto a otro idioma.
+* Extracción de características: crea una representación tensorial del texto.
+
+**Imagen**:
+* Clasificación de imágenes: clasifica una imagen.
+* Segmentación de imágenes: clasifica cada pixel de una imagen.
+* Detección de Objetos: detecta objetos dentro de una imagen.
+
+**Audio**:
+* Clasificación de audios: asigna una etiqueta dado un segmento de audio.
```
Comment (omarespejel) [inline docs/source_es/quicktour.mdx:50]:
```suggestion
* Clasificación de audios: asigna una etiqueta a un segmento de audio.
```

**Review by omarespejel — CHANGES_REQUESTED** [review]
Thanks! Just some nits changes.

**Review by omarespejel — APPROVED** [review]

**Comment by omarespejel** [comment]
Thanks, @Duedme! LGTM!

**Review by sgugger — APPROVED** [review]
Thanks for working on this!
Merging but we won't build the Spanish documentation just yet, we're waiting:
- to have a few more pages in Spanish
- to have the frontend default to the English version of pages when there is no Spanish translation yet (not implemented yet).

I will put a comment here once this page is live (hopefully in one or two weeks!)

**Pull request merged**